### PR TITLE
Add some details + ghost cells

### PR DESCRIPTION
**Pull request by Paulinemoi**

### CODE REVIEW

File: multi-patchs.tex [hunk 1]
```diff
@@ -6,39 +6,109 @@
 
 \section{Structure for multipatch}
 \paragraph{}
+We are trying here to answer to the following questions: 
 \begin{itemize}
-	\item What should be done before September?
+	\item What should be done before September? % TO KEEP ?
```
Comment (alex-m-h) [inline multi-patchs.tex:11]: Good point I think this does not really belong here, right?

File: multi-patchs.tex [hunk 2]
```diff
@@ -6,39 +6,109 @@
...
 	\item What kind of information do the patches need to exchange? 
 \end{itemize}
 
 
 \subsection{Type of global space splitting}
 \paragraph{}
+The splitting of the global space into patches is currently not fixed. However we can encounter three types of sticking: 
+\begin{itemize}
+	\item simple interface which sticks a whole edge on ne dimension; 
+	\item T-joint which sticks interfaces with a partial edge;
```
Comment (alex-m-h) [inline multi-patchs.tex:22]:
```suggestion
	\item simple interface which sticks two whole edges together; 
	\item T-joint which sticks several edges of to one edge of a patch; 
```

File: multi-patchs.tex [hunk 3]
```diff
@@ -121,14 +191,19 @@ \subsection{Type of global space splitting}
...
 \caption{X-point in logical and physical domain.}
 \end{subfigure}
-\caption{Patches in the logical domain.}
+\caption{\label{Patches_logical_dom} Patches in the logical domain.}
 \end{figure}
 
+\newpage
+
+\paragraph{}
+We want here to stay general and to not focus on one patches decomposition of the domain. The figures Fig. \ref{Complex_geometry_1} and \ref{Complex_geometry_2} give example of more complex space.
```
Comment (alex-m-h) [inline multi-patchs.tex:200]:
```suggestion
We want to stay general here and not to focus on one specific decomposition of the domain into patches. The figures Fig. \ref{Complex_geometry_1} and \ref{Complex_geometry_2} give examples of more complex spaces. These are not necessarily for the decomposition of the separatrix.
```

File: multi-patchs.tex [hunk 4]
```diff
@@ -137,17 +212,23 @@ \subsection{Type of global space splitting}
...
 			  Note that the y-dimension of patch {\uppercase\expandafter{\romannumeral 1}}
 			  is periodic.}
 \end{figure*}
+
 \newpage
 
+
 \section{Advection equation}
+\paragraph{}
+We focus on each part of the equations system solving to answer the questions raised in the introduction. We start with the advection equations.
```
Comment (alex-m-h) [inline multi-patchs.tex:230]:
```suggestion
We focus on the advection and Poisson equation separately in the context of the questions raised in the introduction. We start with the advection equation. 
```

File: multi-patchs.tex [hunk 5]
```diff
@@ -156,7 +237,17 @@ \section{Advection equation}
...
+\begin{equation}
+\begin{aligned}
+	\partial_t f 
+		+ v_{GC}\cdot \nabla_{\perp}f 
+		+ v_{\parallel} \partial_z f 
+		+ \dot{v}_{\parallel} \partial_{v_{\parallel}} f
+		= 0,
+\end{aligned}
+\end{equation}
+but it seems to raise the same questions as the guiding center equations.
```
Comment (alex-m-h) [inline multi-patchs.tex:250]: Do you mean that the problems are similar in multipatch geometry?

File: multi-patchs.tex [hunk 6]
```diff
@@ -198,8 +291,8 @@ \section{Advection equation}
 \end{aligned}
 \right.
 \end{equation}
-where $\bar{\psi}$ is a discrete flow computed with
-a time integration method. In the case of Runge-Kutta method, the advection field needs to be evaluated at intermediate feet.
+
+with a time integration method. In the case of Runge-Kutta method, the advection field needs to be evaluated at intermediate feet.
```
Comment (alex-m-h) [inline multi-patchs.tex:295]: Why did you delete the mention of the discrete flow? $\bar{\psi}$ is still undefined

File: multi-patchs.tex [hunk 7]
```diff
@@ -218,17 +311,63 @@ \section{Advection equation}
...
+		\draw[<-] (1.05, 0.58+0.25) -- (1.75, 0.75+0.25) node[above right]{3.}; 
+	\end{scope}
 	
-	\draw[->] (1.5, 0.75) -- (2.78, 0.53) node[below]{feet}; 
-	\draw[<-] (1.5, 1.25) -- (2.78, 1.02) node[above]{value}; 
+	% legend
+	\begin{scope}[xshift=10cm]	
+		\node[right] at (0,1.5) {1. compute feet.};
+		\node[right] at (0,1) {2. evaluate function.};
+		\node[right] at (0,0.5) {3. transfert value.};
```
Comment (alex-m-h) [inline multi-patchs.tex:367]:
```suggestion
		\node[right] at (0,0.5) {3. transfer value.};
```

File: multi-patchs.tex [hunk 8]
```diff
@@ -218,17 +311,63 @@ \section{Advection equation}
...
+	\end{scope}
 \end{tikzpicture}
-\caption{Feet outside the patch in the physical domain.}
+\caption{Example of a feet outside the patch 1 in the logical and physical domains.}
```
[the remark this answers is not in the document]
Comment (alex-m-h) [reply multi-patchs.tex:370]: A very nice sketch!

Comment (alex-m-h) [inline multi-patchs.tex:370]: A very nice sketch!

File: multi-patchs.tex [hunk 9]
```diff
@@ -246,11 +385,10 @@ \section{Advection equation}
 \end{center}
 
 \paragraph{Further questions on outside feet.}
-Problem: The mapping from the logical to the physical domain can be costly to invert. 
+Problem: the mapping from the logical to the physical domain is not always easily invertible. 
 
 \begin{itemize}
-	\item For an advection in the physical domain, if the feet is outside of the domain, how to get the feet in the logical domain of another patch 
-			(inverting the mapping for the other patch might be costly)?
+	\item For an advection in the physical domain, if the feet is outside of the domain, how to get the feet in the logical domain of another patch (could be a not invertible mapping for the other patch)?
```
Comment (alex-m-h) [inline multi-patchs.tex:391]: No, the mappings we consider are always invertible almost everywhere with the only exception of O-points. The problem is that the computation is computationally expensive. Your characteristic foot will not land at the O-point when it leaves a patch, I am pretty sure

File: multi-patchs.tex [hunk 10]
```diff
@@ -357,9 +571,7 @@ \subsection{Further questions}
 	\begin{itemize}
 		\item For a given point in the physical domain, how do we know in which subset it belongs to?
 	\end{itemize}
-	\item Let's assume we know in which patch we are, how can we evaluate a function? 
-			How can we get $\rho(x,y)$ with $(x,y)$ in the physical domain 
-			(especially when the mapping difficult to invert)?
+	\item Let's assume we know in which patch we are, how can we evaluate a function? How can we get $\rho(x,y)$ with $(x,y)$ in the physical domain (especially when the mapping isn't invertible)?
```
Comment (alex-m-h) [inline multi-patchs.tex:573]: Same as above, the mappings are invertible

File: multi-patchs.tex [hunk 11]
```diff
@@ -156,7 +237,17 @@ \section{Advection equation}
 \end{aligned}
 \end{equation}
 
-with $A$ the advection field defined on the logical domain on the physical domain axis. 
+with $A$ the advection field defined on the logical domain on the physical domain axis. Another example is the advection of the drift-kinetic model
```
[the remark this answers is not in the document]
Comment (Paulinemoi) [reply multi-patchs.tex:239]: axes (sorry I thought it was one axe, two axis. But it is one axis, two axes).

File: multi-patchs.tex (hunk 9 above)
Comment (Paulinemoi) [reply multi-patchs.tex:390]: I think I didnt  pull your changes. I didnt wanted to change this sentence.

File: multi-patchs.tex (hunk 8 above)
[the remark this answers is not in the document]
Comment (Paulinemoi) [reply multi-patchs.tex]: "characteristic feet" I prefer, but change done.
